css/main.css

```diff
@@ -325,3 +325,75 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact_form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1vh;
+  padding: 15vh 0 25vh 0;
+  border-bottom: 1px solid black;
+}
+
+.contact_header {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #000;
+}
+
+form {
+  display: flex;
+  gap: 2vh;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#name,
+#email,
+#text {
+  width: 100%;
+}
+
+.label_name,
+.label_email,
+.label_message {
+  padding: 0 10vw;
+}
+
+#name,
+#email {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-color: #d3d0cf;
+  height: 7vh;
+}
+
+#text {
+  background-color: rgb(241, 241, 241);
+  border: none;
+  padding-top: 2vh;
+  padding-left: 1vw;
+}
+
+.label_submit {
+  display: flex;
+  justify-content: center;
+  padding: 2vh 0 5vh 0;
+}
+
+#submit {
+  width: 131px;
+  height: 48px;
+  background: #ff6d04;
+  color: white;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  border-color: #ff6d04;
+}
```

Comment:
- [ ] I noticed that you haven't used the button interactions such as Active, Hover, and Pressed (see Figma design below). Would you please implement them? This makes your User Experience better. 😄 

![image](https://user-images.githubusercontent.com/77190203/147370545-ec44daa3-26b6-4f9b-83e8-8ebb07295ef6.png)